examples/compose/plot_transformed_target.py

```diff
@@ -16,28 +16,13 @@
 # Author: Guillaume Lemaitre <guillaume.lemaitre@inria.fr>
 # License: BSD 3 clause
 
-import numpy as np
-import matplotlib
-import matplotlib.pyplot as plt
-
-from sklearn.datasets import make_regression
-from sklearn.model_selection import train_test_split
-from sklearn.linear_model import RidgeCV
-from sklearn.compose import TransformedTargetRegressor
-from sklearn.metrics import median_absolute_error, r2_score
-from sklearn.utils.fixes import parse_version
-
 # %%
+print(__doc__)
+
+# %% [markdown]
```

Comment:
I'm curious as to what the `[markdown]` tags are used for?

Reply:
It is a jupytext marker to show the paragraph when executing the script in some IDE (like VS code). True that this might not be necessary.